pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -2694,6 +2694,15 @@ package-maintainers:
     # - pipes-mongodb
     - streaming-wai
   kiwi:
+    - config-schema
+    - config-value
+    - glirc
+    - irc-core
+    - matterhorn
+    - mattermost-api
+    - mattermost-api-qc
+    - QuickCheck_2_14_2
+    - random_1_2_0
```

Comment:
This PR looks good, although since these packages will probably get updated at some point, I'm not sure if it would be okay to keep them here.

I'll remove them just to be safe.

```suggestion
```